server/boards/queries.ts

```diff
@@ -426,3 +426,23 @@ export const getBoardRoles = async ({
   });
   return boardRoles;
 };
+
+export const deleteBoard = async ({
+  boardExternalId,
+  firebaseId,
```

Comment:
You shouldn't need the `firebaseId` here with the bare-bones hard delete you're implementing as that's the id of the user. (If we we're doing a soft delete it might make sense to record who deleted it. )